### ISSUE TITLE
[FEATURE REQUEST] more code data logger updates!

### ISSUE THREAD

**Issue by g0me3** [TASVideos/BizHawk#1738]
ok, now all updates just fine as usual. but then I'd like to request more, sorry ;)
is it possible to keep windows updating while "TURBO" is pressed?
this one is make me mad me sometimes. need to release turbo button to see if something changed and there are a lot of this sometimes. (also that reminds me that I must have been put another feature request about to make turbo button togglable not only pushable).

**Comment by zeromus** [comment]
Isn't the definition of turbo that it doesn't update anything, so that emulation can go as fast as possible? I don't think it's a great idea (even if it's possible) to start specially hacking certain tool windows to get different and seemingly arbitrarily chosen turbo handling, or to start adding options to tool windows to control that (although the tool window framework could help with that...). It isn't the worst idea in the world either, but the whole concept may need a little rethinking.

**Comment by vadosnaprimer** [comment]
You can increase emulation speed or disable throttling altogether via Config - Speed/Skip.

**Comment by g0me3** [comment]
you see, turbo may be very helpful when you need to speed up the game to not to wait too long for something is changing. but sometimes its is hard to tell if something is changed when in turbo mode at all, so I need to disable it to check this. so I need turbo to be off when i need turbo to be on very often...  the option to force window update in any modes or either remove certain window from turbo disable list would be helpful..

and yes, any other speed / throttling options are not the same as turbo and always WAY slower not because all windows are updated, but just by method they used to speedup things...

**Comment by ehw** [comment]
Not sure what this has to do with the CDL, this seems to be for something else entirely...

But it would be cool to get CDL support on PicoDrive so you can log 32x games. 32x games are tricky to analyze with a disassembler since most ROMs consist of data for three different CPUs, not including the VDP. CDL support for 32x (and even N64, despite how resource intensive that might be) would be great. :)

**Comment by g0me3** [comment]
yes, this is exactly 38000 at cdl file and i have here like this
https://www.dropbox.com/s/9khfpsdpeue921t/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%202020-01-22%2023.05.46.png?dl=0

**Comment by YoshiRulz** [comment]
Re-reading this thread, and yeah you can just set the target speed in any of the mentioned ways.
Being able to switch keybinds between toggle and hold mode is a nice-to-have that we'll look at as part of our [keybinds rewrite](https://gitlab.com/YoshiRulz/yoshis-hawk-thoughts/-/issues/27) (#1925/#2981).
